internal/util/importutilv2/parquet/field_reader.go

```diff
@@ -353,8 +359,12 @@ func ReadNullableIntegerOrFloatData[T constraints.Integer | constraints.Float](p
 			for i := 0; i < dataNums; i++ {
 				data = append(data, T(float64Reader.Value(i)))
 			}
+		case arrow.NULL:
+			// the chunk type may be *array.Null if the data in chunk is all null
+			validData = append(validData, make([]bool, dataNums)...)
+			data = append(data, make([]T, dataNums)...)
 		default:
-			return nil, nil, WrapTypeErr("integer|float", chunk.DataType().Name(), pcr.field)
+			return nil, nil, WrapTypeErr("integer|float or null", chunk.DataType().Name(), pcr.field)
```

Comment:
please help to format this to `integer or float or null` or `integer|float|null`